### PR TITLE
Set $VERSION env var

### PR DESCRIPTION
**Pull request by justcalamari**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Not for this PR but I guess we should test everything but pushing for the migrators.

File: conda_forge_tick/migrators.xsh [hunk 1]
```diff
@@ -224,16 +224,19 @@ class Version(Migrator):
         return body
 
     def commit_message(self):
-        return "updated v" + $VERSION
```
Comment (isuruf) [inline conda_forge_tick/migrators.xsh]: How about just replacing $VERSION with self.version here?

**Comment by CJ-Wright** [comment]
One thing we need to make sure of is that we make sure that version is re-set for every single migration. We don't want bleed between the recipes.

**Comment by justcalamari** [comment]
> One thing we need to make sure of is that we make sure that version is re-set for every single migration. We don't want bleed between the recipes.

The first line in `migrate` is `self.version = attrs['new_version']`, so I think this is okay.

**Comment by scopatz** [comment]
LGTM!

**Comment by CJ-Wright** [comment]
Thank you very much @justcalamari

**Pull request merged**